web-client/src/presenter/actions/startRefreshIntervalAction.test.js

```diff
@@ -33,7 +33,7 @@ describe('startRefreshIntervalAction', () => {
     ).toHaveBeenCalled();
     expect(
       applicationContext.getUseCases().setItemInteractor,
-    ).toHaveBeenCalled();
+    ).not.toHaveBeenCalled();
```

Comment:
Should we add a test with `process.env.IS_LOCAL` set to `true` and therefore have an expect for having `setItemInteractor` to have been called?